### PR TITLE
Update deployer to newer version

### PR DESCRIPTION
**Pull request by tshabatyn**
The old version of deployer throws the following error in the Docker container.
```
PHP Fatal error:  require(): Failed opening required 'recipe/common.php' (include_path='phar:///var/www/html/bin/dep/bin/../:.:/usr/share/pear:/usr/share/php') in /var/www/html/vendor/jalogut/magento2-deployer-plus/recipe/magento_2_2_5.php on line 13
```

### CODE REVIEW

**Comment by erfanimani** [comment]
LGTM, happy to merge this in. What do you think @sprankhub?

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+  - 7.4
 
 before_install:
   - sudo apt-get -qq update
 
 install:
-  - composer self-update
+  - php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+  - php -r "if (hash_file('sha384', 'composer-setup.php') === '756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+  - php composer-setup.php --version=1.10.19
```
Comment (sprankhub) [inline .travis.yml:14]:
```suggestion
  - php composer-setup.php --1
```

Why do we fix this to a specific version? I suggest using at least the latest v1 version. Or can we even switch to v2?

**Review by sprankhub — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (erfanimani) [reply .travis.yml:14]: I've been using this with Composer 2 just fine, so it probably could do without a version

Comment (PauloPhagula) [reply .travis.yml:14]: Indeed, composer 2 is now supported on both 2.3 and 2.4, so I agree that there's no need to hardcore composer version to v1

**Comment by erfanimani** [comment]
Closing this one as I've created a newer PR which includes this one (https://github.com/jalogut/magento2-deployer-plus/pull/80)

**Pull request closed without merging**